### PR TITLE
[Tools] Enable --gas-limit for wasmedge cli

### PR DESCRIPTION
**Pull request by hydai**
Signed-off-by: hydai <hydai@secondstate.io>

### CODE REVIEW

**Comment by juntao** [comment]
Let's deprecate pages like `docs/run.md` and use the `book` as the single source of truth. Thanks.

https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/start/cli.md

**Comment by hydai** [comment]
> Let's deprecate pages like `docs/run.md` and use the `book` as the single source of truth. Thanks.
> 
> https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/start/cli.md

Sure. Should I also redirect these pages to the new one?

**Comment by juntao** [comment]
I do not think GitHub will do http redirect? Perhaps we can remove the content in the md file and put a link to the book's html page there for people to click on?

**Comment by hydai** [comment]
That's what I want. Put a link.
However, I am not sure which one is better: put a link to the webpage or to the new markdown file?

**Comment by juntao** [comment]
> That's what I want. Put a link. However, I am not sure which one is better: put a link to the webpage or to the new markdown file?

I think web page is better since the visitor is typically looking to read this content. The HTML page has an "edit" button that brings the reader directly to the MD source file.

File: include/common/statistics.h [hunk 1]
```diff
@@ -131,6 +134,40 @@ class Statistics {
            TimeRecorder.getRecord(Timer::TimerTag::HostFunc);
   }
 
+  void dumpToLog(const Configure &Conf) const noexcept {
+    auto Nano = [](auto &&Duration) {
+      return std::chrono::nanoseconds(Duration).count();
+    };
+    if (Conf.getStatisticsConfigure().isTimeMeasuring() ||
```
Comment (ibmibmibm) [inline include/common/statistics.h:141]: `const auto &StatConf = Conf.getStatisticsConfigure();`

**Review by ibmibmibm — CHANGES_REQUESTED** [review]

**Review by ibmibmibm — APPROVED** [review]

**Pull request merged**